### DISCUSSION TITLE
Katas 10/02

### DISCUSSION THREAD

**Discussion by Mich-Arv** [LaunchX-InnovaccionVirtual/CursoIntroPython#113] [General]
Hola!
¿Ustedes tienen activas los recursos de los últimos 4 módulos del curso?

![image](https://user-images.githubusercontent.com/25958583/153534857-6435b868-dfaa-433d-a723-ef0a80d2953a.png)

**Comment by BarajasAngel** [comment]
No, al parecer lo habilitaran a las 10 sin embargo puede que tarden un poco mas, solo queda esperar
